woocommerce/woocommerce
# E2E: add registered shopper checkout test

### All Submissions:


### Changes proposed in this Pull Request:

Closes #27883. 

### How to test the changes in this Pull Request:

1. check out branch locally
2. [run end-to-end tests](https://github.com/woocommerce/woocommerce/tree/master/tests/e2e#running-tests) and ensure they pass

### Other information:


### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

N/A
